### PR TITLE
assorted typo fixes

### PR DESCRIPTION
**Pull request by vp89**
Just thought I would help out by fixing a few small typos I noticed while reading through the content.

This is very high quality stuff! Definitely should be compiled into a book eventually IMO.

Looking forward to more posts especially in the "Operating" category ... "Curating technical quality" seems like an interesting topic.

### CODE REVIEW

File: src/markdown/guides/systems-thinking.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ They focus on four measures of developer velocity:
 1.  **Time to restore service** is time spent recovering from defects.
 
 The book uses surveys from tens of thousands of organizations to assess their
-overall producitivty and show how it correlates to their performance on those
+overall producitivity and show how it correlates to their performance on those
```
Comment (vp89) [inline src/markdown/guides/systems-thinking.md:43]:
```suggestion
overall productivity and show how it correlates to their performance on those
```

**Comment by lethain** [comment]
@vp89 - wow, I just wanted to say thank you for taking the time to make all these improvements, this is a major overall improvement to the content and I really appreciate it

**Comment by lethain** [comment]
@vp89 -- oh, and definitely working on compiling it into a book, hopefully early 2021!

**Comment by lethain** [comment]
Build and deployed, staffeng.com is updated (CDN might cache for a minute or two)

**Pull request merged**